resources/lang/en/beatmaps.php

```diff
@@ -67,15 +67,42 @@
         ],
 
         'message_type' => [
-            'disqualify' => 'Disqualify',
```

Comment:
this should be done in a way that doesn't remove all existing translations. Add a suffix, use different group, etc.

Reply:
resolved by using a different group